elisp.nix

```diff
@@ -40,6 +40,8 @@ let
       else if type == "string" then config
       # - A config path { config = ./config.el; }
       else if type == "path" then builtins.readFile config
+      # - A list of paths { config = [ ./config.el ./custom.el ] }
+      else if type == "list" then lib.concatMapStringsSep "\n" (fname: builtins.readFile fname) config
```

Comment:
I am slightly against this because the check `type == "list"` is weaker that `a list of paths`.

Reply:
Yep, agreed, I suppose a check of what the list elements actually are is possible.